packages/lesswrong/lib/rateLimits/utils.ts

```diff
@@ -154,15 +125,23 @@ export function calculateRecentKarmaInfo(userId: string, allVotes: RecentVoteInf
   }
 }
 
+export function calculateFeaturesForTest(userId: string, userKarmaInfo: UserKarmaInfo, allVotes: RecentVoteInfo[]): AnyBecauseTodo {
```

Comment:
Code exclusively used for tests should live in the test file.  Also, isn't the return type just `RateLimitFeatures`?